### PR TITLE
Updated visual-studio-code/index file

### PR DESCRIPTION
**Pull request by jm-trd-ms**
Updated content in visual-studio-code/index.md. 

Updated figure and metadata.

### CODE REVIEW

**Review by Reezaali — CHANGES_REQUESTED** [review]
Can we remove the code comments and TODOs please?

**Comment by mikehoffms** [comment]
@codepo8 Please review this PR and Approve it if it's a sufficient improvement for now.  This PR was apparently intended to just make a couple spot-fixes to update a screenshot.

"Visual Studio Code overview" (microsoft-edge/visual-studio-code/index.md): 
[Before](https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/)
[After](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/?branch=pr-en-us-1053)

"Microsoft Edge Developer Tools extension for Visual Studio Code" (microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md)
[Before](https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension)
[After](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension?branch=pr-en-us-1053)

"The webhint extension for Visual Studio Code" (microsoft-edge/visual-studio-code/webhint.md)
[Before](https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/webhint)
[After](https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/webhint?branch=pr-en-us-1053)

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 1]
```diff
@@ -1,188 +1,220 @@
...
-:::image type="complex" source="./media/edge-devtools-for-vscode-elements.png" alt-text="Elements tool open with HTML highlighted" lightbox="./media/edge-devtools-for-vscode-elements.png":::
-   Elements tools open with HTML highlighted  
+:::image type="complex" source="./media/edge-devtools-for-vscode-elements.png" alt-text="Elements tool with HTML highlighted" lightbox="./media/edge-devtools-for-vscode-elements.png":::
+   The **Elements** tool with HTML highlighted
 :::image-end:::
 
-## Using Microsoft Edge Developer Tools in Visual Studio Code  
 
+<!-- ====================================================================== -->
+## Modes for using Microsoft Edge Developer Tools in Visual Studio Code
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:64]: @codepo8 How do these modes relate to the headings?  Does each of these 3 modes (list items) correspond to a specific heading below?  If so, which heading is for which mode that's listed here?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 2]
```diff
@@ -1,188 +1,224 @@
...
+:::image type="complex" source="./media/edge-devtools-for-vscode-elements.png" alt-text="Elements tool with HTML highlighted" lightbox="./media/edge-devtools-for-vscode-elements.png":::
+   The **Elements** tool with HTML highlighted
 :::image-end:::
 
-## Using Microsoft Edge Developer Tools in Visual Studio Code  
 
+<!-- ====================================================================== -->
+## Modes for using Microsoft Edge DevTools in Visual Studio Code
+
+<!-- todo: related this list to the subsequent sections -->
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:69]: @codepo8 - Re-entering comment (2nd instance) b/c "Outdated".
Which list item corresponds to which heading below, if any?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 3]
```diff
@@ -195,33 +231,44 @@ JavaScript debugging is now built in to Visual Studio Code. Starting with Versio
...
+    ```
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-console-integration.png" alt-text="Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow" lightbox="./media/edge-devtools-for-vscode-console-integration.png":::
-   Microsoft Edge Developer Tools Console available when extension launched from Run and Debug workflow  
-:::image-end:::
+1.  Change `http://localhost:8080` in the above code and make sure the variable `{workspaceFolder}` resolves.
 
-## Getting in touch with the Microsoft Edge DevTools for Visual Studio Code extension team  
+1.  Select the **Inspect** icon.  If you haven't installed the Microsoft Edge DevTools extension for Visual Studio Code, the **Extensions** tab opens and automatically displays the extension to install. <!-- this step was stray; move? -->
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:238]: @codepo8 - this step is stray, it was in the middle of the above step/paragraph.  Move up?

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md [hunk 4]
```diff
@@ -195,35 +235,46 @@ JavaScript debugging is now built in to Visual Studio Code. Starting with Versio
...
-<!--links -->  
 
-[VisualstudioCode]: https://code.visualstudio.com "Visual Studio Code"  
-[VisualStudioCodeDocs]: https://code.visualstudio.com/Docs "Documentation | Visual Studio Code"   
+<!-- ====================================================================== -->
+## Console integration
+
+If you launch the extension from the Run and Debug workflow, the [Debug Console of Visual Studio Code][DebugConsoleVSCode] gives you all the functions of the DevTools [Console][EdgeDevToolsConsole].  You have direct access to the Window object of the instance of Edge and can use all the [Console Utilities][ConsoleUtilities].
+
+:::image type="complex" source="./media/edge-devtools-for-vscode-console-integration.png" alt-text="The DevTools Console is available when the extension is launched from a Run and Debug workflow" lightbox="./media/edge-devtools-for-vscode-console-integration.png":::
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:254]: This image is too small.  Re-create or find a bigger original.

Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:254]: @codepo8 - Do you have bigger versions of the .png files for this article?  They are somewhat too small, and clicking the images ("lightbox" view) doesn't make them bigger.

**Review by codepo8 — APPROVED** [review]
This looks good to me.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:242]: Not very harmful, at end of section.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md]: Nice to-do for a later PR.

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md]: dup (a result of github PR ui)

File: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md:254]: Notified writer to add article to list to update screenshots.

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**